lib/msal-common/test/telemetry/PerformanceClient.spec.ts

```diff
@@ -3,9 +3,16 @@
  * Licensed under the MIT License.
  */
 
-import { ApplicationTelemetry, IGuidGenerator, IPerformanceMeasurement, Logger, PerformanceEvents } from "../../src";
-import { IPerformanceClient } from "../../src/telemetry/performance/IPerformanceClient";
-import { PerformanceClient } from "../../src/telemetry/performance/PerformanceClient";
+import {
```

Comment:
Import from the relevant file(s), not src

Reply:
Good catch, thank you Thomas. Fixed.